### DISCUSSION TITLE
[Feature] Improved memories

### DISCUSSION THREAD

**Discussion by fredsco** [immich-app/immich#11484] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently memories show on the top with the text "x years ago" It would be great to have memories that are more contextual. E.g "Trip in {Location}"  or "Sunday evening at {museum name}"

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7437
